Makefile.defs

```diff
@@ -165,22 +167,15 @@ else ifeq ($(TARGET), appimage)
 	export EMULATE_READER=1
 	export SDL=1
 	export APPIMAGE=1
-else ifeq ($(TARGET), debian)
-	export SDL=1
-	export DEBIAN=1
-	export EMULATE_READER=1
-else ifeq ($(TARGET), debian-armel)
-	CHOST?=arm-linux-gnueabi
+else ifeq ($(TARGET), linux)
 	export SDL=1
-	export DEBIAN_CROSS=1
-else ifeq ($(TARGET), debian-armhf)
-	CHOST?=arm-linux-gnueabihf
-	export SDL=1
-	export DEBIAN_CROSS=1
-else ifeq ($(TARGET), debian-arm64)
-	CHOST?=aarch64-linux-gnu
-	export SDL=1
-	export DEBIAN_CROSS=1
+	ifeq ($(LINUX_ARCH), arm)
+		CHOST?=arm-linux-gnueabihf
+	else ifeq ($(LINUX_ARCH), arm64)
+		CHOST?=aarch64-linux-gnu
+	else
+		export EMULATE_READER=1
```

Comment:
Mainly a note to self not to merge just yet, I didn't fully check what needs to be changed. :-)
```suggestion
		LINUX=1
		export EMULATE_READER=1
```